### PR TITLE
[issue29] update footer style

### LINKED ISSUE #29: update footer styling
**Is your feature request related to a problem? Please describe.**
less padding to the footer because it looks like it has more priority than the content.

**Describe how the solution should behave**

**Screenshots of the design/if any**
Untitled

**Additional context**
**Issue**
footer issue

### PR DESCRIPTION
**Pull request by RawandDev**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
 function Footer() {
   return (
     <footer>
-      <div className="p-10 bg-gray-200 text-black">
+      <div className="px-10 bg-gray-200 text-black">
```
Comment (Niyaz-Dev12) [inline src/components/Footer.jsx]: can you add a pt-1 too.

**Review by Niyaz-Dev12 — COMMENTED** [review]
LGTM. just one note

File: src/components/Footer.jsx (hunk 1 above)
Comment (RawandDev) [reply src/components/Footer.jsx]: Done. Thank you.

**Review by MuhannedNoman — DISMISSED** [review]
LGTM

**Comment by MuhannedNoman** [comment]
This looks like an issue now 
image

**Comment by RawandDev** [comment]
Need review please.

**Comment by MuhannedNoman** [comment]
There is an extra bottom white space in Mobile and desktop view.

**Comment by MuhannedNoman** [comment]
To be honest, the footer is working perfectly by default. This PR is just messing things up.

**Comment by RawandDev** [comment]
> There is an extra bottom white space in Mobile and desktop view.

I think it is because the content is small and cant fill the height.

**Comment by RawandDev** [comment]
> To be honest, the footer is working perfectly by default. This PR is just messing things up.

Ok. I will try to close this later.

**Comment by RawandDev** [comment]
Closing this pull request since it had some issues.

**Pull request closed without merging**